.test/config_single_end/config.yaml

```diff
@@ -0,0 +1,74 @@
+# This file should contain everything to configure the workflow on a global scale.
+# In case of sample based data, it should be complemented by a samples.tsv file that contains
+# one row per sample. It can be parsed easily via pandas.
+samples: "config_single_end/samples.tsv"
+# to download reads from SRA the accession numbers (see https://www.ncbi.nlm.nih.gov/sra) of samples must be given in
+# units.tsv dataset for testing this workflow with single end reads:
+# https://www.ncbi.nlm.nih.gov/Traces/study/?acc=PRJNA255509&o=acc_s%3Aa
+units: "config_single_end/units.tsv"
+single_end: True
+
+# config for a small single end data set for testing
+resources:
+  ref:
+    # Number of chromosomes to consider for calling.
+    # The first n entries of the FASTA will be considered.
+    n_chromosomes: 25
+    # Ensembl species name
+    species: homo_sapiens
+    # Ensembl release
+    release: 101
+    # Genome build
+    build: GRCh38
+    # for testing data a specific chromosome can be selected
+    chromosome:
+    # specify release version number of igenomes list to use (see https://github.com/nf-core/chipseq/releases), default: 1.2.2
+    igenomes_release: 1.2.2
+    # if igenomes.yaml cannot be used, a value for the mappable or effective genome size can be specified here, e.g. macs-gsize: 2.7e9
+    macs-gsize:
+    # if igenomes.yaml cannot be used, a path to an own blacklist can be specified here
+    blacklist:
+
+params:
+  # choose "narrow" or "broad" for macs2 callpeak analysis, for documentation and source code please see https://github.com/macs3-project/MACS
+  peak-analysis: "broad"
+  # Number of biological replicates required from a given condition for a peak to contribute to a consensus peak
+  min-reps-consensus: 1
+  callpeak:
+    p-value: 0.5
+    q-value:
+  deeptools-plots:
+    # when activated the plot profile and heatmap plot are generated, this involves a matrix calculation that requires a lot of working memory.
+    activate: False
```

Comment:
The data are significantly reduced for the single end test, but they are still very large. The matrix calculation is only done to generate 2 plots, without further use in the workflow. Since the matrix calculation on the large data takes a long time, this usually leads to a timeout with github actions. Since this matrix and the two plots are also calculated on the much smaller paired end data, one test (the paired end test) is actually sufficient to perform this. I would therefore like to leave it at False for the single end data test, how do you see this?

Reply:
But this is not for the reduced single end test, but for the one that we hope to run on a dedicated testing server in the future, right?

Reply:
indeed, you are right...thanks :)